tests/test_plot_sample_hist.py

```diff
@@ -26,25 +26,25 @@ def test_plot_sample_hist():
     # tests exception is raised when 'pop' or 'samples' is not a dataframe
     with pytest.raises(TypeError) as e:
       assert plot_sample_hist([1, 2, 3], samples)
```

Comment:
I think now we don't need `e`'s anywhere (this was my mistake, sorry!)

```suggestion
    with pytest.raises(TypeError):
      assert plot_sample_hist([1, 2, 3], samples)
```